doc/src/explore/examples.md

```diff
@@ -11,6 +11,10 @@ Here is a rundown of existing examples in the Sui repository.
 
 In the [Basics](https://github.com/MystenLabs/sui/tree/main/sui_programmability/examples/basics) example, explore object creation, update, and exchange.
 
+## Crypto
+
+In the [Cryptography](https://github.com/MystenLabs/sui/tree/main/sui_programmability/examples/math) example, employ a simple contract to perform verification and ecrecover using a [Secp256k1](https://crates.io/crates/secp256k1/) curve and hash with keccak256 and producing an event with data as a result.
```

Comment:
thank you! I made some changes to the example contract implementation:

```
* ECDSA: A simple contract to do the following: 
1) Hash a piece of data using keccak256 and output an object with hashed data.
2) Recover a Secp256k1 signature to its public key, output an object with the public key. 
3) Verify a Secp256k1 signature, produce an event for whether it is verified. 
``` 
Feel free to repurpose the language and link to the example https://github.com/MystenLabs/sui/pull/4262